pkgs/os-specific/linux/uhk-agent/default.nix

```
@@ -0,0 +1,38 @@
+{ appimageTools, lib, fetchurl, polkit, udev }:
+let
+  pname = "uhk-agent";
+  version = "1.5.16";
+  name = "${pname}-${version}";
+  src = fetchurl {
+    url = "https://github.com/UltimateHackingKeyboard/agent/releases/download/v${version}/UHK.Agent-${version}-linux-x86_64.AppImage";
+    name = "${name}.AppImage";
+    sha256 = "sha256-nJxfhmHt2bnu+lQOL7zQPzBe03n6yAVtyG7Y1atOpgg=";
+  };
+
+  appimageContents = appimageTools.extract {
+    name = "${pname}-${version}";
+    inherit src;
+  };
+in appimageTools.wrapType2 {
+  inherit pname src name;
```

```suggestion
  inherit name pname version src;
```

I think we need this to make ``uhk-agent.version`` work.

---

I added the version to the meta attributes of uhk-agent. Now I can set `version = uhk-agent.meta.version;` in uhk-udev-rules/default.nix.

Hope this this okay for your.